internal/pgbackrest/config.go

```diff
@@ -172,6 +188,8 @@ func MakePGBackrestLogDir(template *corev1.PodTemplateSpec,
 		}
 	}
 	template.Spec.InitContainers = append(template.Spec.InitContainers, container)
+
+	return pgBackRestLogPath
```

Comment:
I got tired of looping through repos to get this name, so I had this func return it since I needed this name right after this func was called. I _could've_ done that work outside this func or create a util, maybe I'll consider that tomorrow.